payload-files/nextjs-13_2_0/NextServer.getRequestHandler--transaction.json

```diff
@@ -0,0 +1,320 @@
+      "estree-walker": "2.0.2",
+      "fs.realpath": "1.0.0",
+      "function-bind": "1.1.2",
+      "has-flag": "4.0.0",
+      "hasown": "2.0.2",
+      "import-in-the-middle": "1.7.1",
+      "inflight": "1.0.6",
+      "inherits": "2.0.4",
+      "is-core-module": "2.13.1",
+      "is-reference": "1.2.1",
+      "js-tokens": "4.0.0",
+      "loose-envify": "1.4.0",
+      "magic-string": "0.27.0",
+      "module-details-from-path": "1.0.3",
+      "next": "13.2.0",
+      "once": "1.4.0",
+      "opentelemetry-instrumentation-fetch-node": "1.2.0",
+      "picomatch": "2.3.1",
+      "react": "18.2.0",
+      "react-dom": "18.2.0",
+      "require-in-the-middle": "7.3.0",
+      "resolve": "1.22.8",
+      "rollup": "3.29.4",
+      "scheduler": "0.23.0",
+      "semver": "7.6.0",
+      "shimmer": "1.2.1",
+      "stacktrace-parser": "0.1.10",
+      "styled-jsx": "5.1.1",
+      "supports-color": "5.5.0",
+      "wrappy": "1.0.2",
+      "yallist": "4.0.0"
+    },
+    "platform": "node",
+    "sdk": {
+      "integrations": [
+        "InboundFilters",
+        "FunctionToString",
+        "LinkedErrors",
+        "RequestData",
+        "Console",
+        "NodeFetch",
+        "OnUncaughtException",
+        "OnUnhandledRejection",
+        "ContextLines",
+        "LocalVariablesAsync",
+        "Context",
+        "Modules",
+        "Express",
+        "Fastify",
+        "Graphql",
+        "Mongo",
+        "Mongoose",
+        "Mysql",
+        "Mysql2",
+        "Postgres",
+        "Nest",
+        "Hapi",
+        "Koa",
+        "Connect",
+        "Http",
+        "DistDirRewriteFrames"
+      ],
+      "name": "sentry.javascript.nextjs",
+      "packages": [
+        {
+          "name": "npm:@sentry/nextjs",
+          "version": "8.0.0-beta.3"
+        },
+        {
+          "name": "npm:@sentry/node",
+          "version": "8.0.0-beta.3"
+        }
+      ],
+      "version": "8.0.0-beta.3"
+    },
+    "server_name": "D9M3PY4LQ7.local",
+    "spans": [
+      {
+        "data": {
+          "otel.kind": "INTERNAL",
+          "sentry.origin": "manual"
+        },
+        "description": "BaseServer.renderToResponse",
+        "origin": "manual",
+        "parent_span_id": "[[ID3]]",
+        "span_id": "[[ID4]]",
+        "start_timestamp": "[[timestamp]]",
+        "status": "ok",
+        "timestamp": "[[timestamp]]",
+        "trace_id": "[[ID2]]"
+      },
+      {
+        "data": {
+          "otel.kind": "INTERNAL",
+          "sentry.origin": "manual"
+        },
+        "description": "NextNodeServer.findPageComponents",
+        "origin": "manual",
+        "parent_span_id": "[[ID4]]",
+        "span_id": "[[ID5]]",
+        "start_timestamp": "[[timestamp]]",
+        "status": "ok",
+        "timestamp": "[[timestamp]]",
+        "trace_id": "[[ID2]]"
+      }
+    ],
+    "start_timestamp": "[[timestamp]]",
+    "timestamp": "[[timestamp]]",
+    "transaction": "NextServer.getRequestHandler",
```

Comment:
A lot of events/transaction in Next 13.2.0 don't have the correct transaction name (e.g. `GET /api/test-error-manual`). The name is `NextServer.getRequestHandler` or `BaseServer.renderToResponse`. Hence, those two .json files are created. The json files are named after `transaction` or `url`. Normally, we can use `url`, but the Next.js output does not provide `url` at the moment.